src/cdk/coercion/coercion.md

```diff
@@ -1,3 +1,59 @@
-### Coercion
-
 Utility functions for coercing `@Input`s into specific types.
+
+### Example
+
+```ts
+import {Directive, ElementRef, HostListener} from '@angular/core';
+import {
+  coerceBooleanProperty,
+  BooleanInput,
+  NumberInput,
+  coerceNumberProperty,
+  coerceElement,
+  } from '@angular/cdk/coercion';
+
+@Directive({
+  selector: 'my-button',
+  host: {
+    '[disabled]': 'disabled'
+  }
+})
+class MyButton {
+  // Using `coerceBooleanProperty` allows for the disabled value of a button to be set as
+  // `<my-button disabled></my-button>` instead of `<my-button [disabled]="true"></my-button>`.
+  // It also allows for a string to be passed like `<my-button disabled="true"></my-button>`.
+  @Input()
+  get disabled() { return this._disabled; }
+  set disabled(value: any) {
+    this._disabled = coerceBooleanProperty(value);
+  }
+  private _disabled = false;
+
+  // Using `coerceNumberProperty` allows for any value to be set to `greetDelay` while ensuring
+  // that a number is always returned. E.g. the consumer can write:
+  // `<my-button greetDelay="500"></my-button>` instead of `<my-button [greetDelay]="500"></my-button>`.
```

Comment:
I wonder if most users will see that `greetDelay="500"` means its passing a string, and that's issue the coercion is solving. Could we add a sentence explaining that its turning the string into a number?